src/sentry/south_migrations/0421_auto__add_grouphashtombstone__add_unique_grouphashtombstone_project_ha.py

```diff
@@ -0,0 +1,1189 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        # Removing unique constraint on 'IdentityProvider', fields ['type', 'organization_id', 'external_id']
+        db.delete_unique(u'sentry_identityprovider', ['type', 'organization_id', 'external_id'])
```

Comment:
@adhiraj Hey I think this is related to your work. I auto-generated this migration and this was included. Was something wrong/missed in a previous migration?

Reply:
@bretthoerner this is intentional, but we’re trying to do a series of like, 5 migrations and it’s taking longer than I anticipated. Do you mind waiting until all of his other bits are merged? Then things will be good. When deleting a column, we have to spread it across a few deploys. And the first step involves removing it from code, without actually deleting the column from the database to avoid errors.

Reply:
I think this is the last bit we need, which is what will conflict here: https://github.com/getsentry/sentry/pull/8615

Reply:
Sure, I can update the migration again later. Do you think this is happening today or... longer?

Reply:
I'll do it today morning